src/components/organisms/Footer/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import './Footer.scss';
+
+function Footer() {
+    const date = new Date();
```

Comment:
jeśli nie zamierzasz używać tego obiektu więcej niż ten jeden raz do pobrania roku, to przypisałbym tutaj do zmiennej już ten rok, a niżej tylko pobrał tę jedną zmienną.